src/components/Reports/ResponsibilityCentersReport/ResponsibilityCentersReport.tsx

```diff
@@ -207,6 +213,11 @@ export const ResponsibilityCentersReport: React.FC<Props> = ({
         />
       ) : (
         <ScrollBox data-testid="ResponsibilityCentersScrollBox">
+          <Alert severity="warning">
+            {t(
+              'Responsibility Centers is still in development. Some features are not working. Please check back later.',
```

Comment:
I would say it should be coming in the next few days to give people hope. Sometimes Companies say a feature is coming soon and it takes weeks/months.